src/site/content/en/blog/css-marker-pseudo-element/index.md

```diff
@@ -0,0 +1,302 @@
+    alt="List Demo on Glitch"
+    style="height: 100%; width: 100%; border: 0;">
+  </iframe>
+</div>
+
+{% Aside 'gotchas' %}
+In Chromium, `white-space` only works for inside positioned markers. For outside positioned markers, the style adjuster always forces `white-space: pre` in order to preserve the trailing space.
+{% endAside %}
+
+
+#### Changing the content of a marker
+This CSS property can be used in a few ways and we felt it was important to cover a few. 
+
+**Changing all list items**
+```css
+li {
+  list-style-type: "😍";
+}
+
+/* OR */
+
+li::marker {
+  content: "😍";
+}
+```
+
+<div class="glitch-embed-wrap" style="height: 420px; width: 100%;">
+  <iframe
+    src="https://glitch.com/embed/#!/embed/marker-change-all?path=style.css&previewSize=100"
+    alt="List Demo on Glitch"
+    style="height: 100%; width: 100%; border: 0;">
+  </iframe>
+</div>
+
+**Changing just 1 list item**
+```css
+li:last-child::marker {
+  content: "😍";
+}
+```
+
+<div class="glitch-embed-wrap" style="height: 420px; width: 100%;">
+  <iframe
+    src="https://glitch.com/embed/#!/embed/marker-change-one?path=style.css&previewSize=100"
+    alt="List Demo on Glitch"
+    style="height: 100%; width: 100%; border: 0;">
+  </iframe>
+</div>
+
+**Changing a list item to SVG**
+```css
+li::marker {
+  content: url(/heart.svg);
+  content: url(#heart);
+  content: url("data:image/svg+xml;charset=UTF-8,<svg xmlns='http://www.w3.org/2000/svg' version='1.1' height='24' width='24'><path d='M12 21.35l-1.45-1.32C5.4 15.36 2 12.28 2 8.5 2 5.42 4.42 3 7.5 3c1.74 0 3.41.81 4.5 2.09C13.09 3.81 14.76 3 16.5 3 19.58 3 22 5.42 22 8.5c0 3.78-3.4 6.86-8.55 11.54L12 21.35z' fill='none' stroke='hotpink' stroke-width='3'/></svg>");
+}
+```
+
+<div class="glitch-embed-wrap" style="height: 420px; width: 100%;">
+  <iframe
+    src="https://glitch.com/embed/#!/embed/marker-inline-svg?path=style.css&previewSize=100"
+    alt="List Demo on Glitch"
+    style="height: 100%; width: 100%; border: 0;">
+  </iframe>
+</div>
+
+**Changing numbered lists**  
+What about an `<ol>` though right? Each of those `::markers` aren't the simple bullet, they're numbers. In CSS these are called [Counters](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Lists_and_Counters/Using_CSS_counters), and they're quite powerful. They even have properties to set and reset where the number starts and ends, or switching them to roman numerals. Can we style that? Yep, and we can even use the marker content value to build our own numbering presentation.
+
+```css
+li::marker {
+  content: counter(list-item) "› "; 
+  color: hotpink;
+}
+```
+
+<div class="glitch-embed-wrap" style="height: 420px; width: 100%;">
+  <iframe
+    src="https://glitch.com/embed/#!/embed/marker-numbered-lists?path=style.css&previewSize=100"
+    alt="List Demo on Glitch"
+    style="height: 100%; width: 100%; border: 0;">
+  </iframe>
+</div>
```

Comment:
It might be worthwhile to add some feature detection or warning text to the demo saying that this won't work unless you're on the correct version of Chrome, because this is what I saw as I scanned through the content

![image](https://user-images.githubusercontent.com/4713486/88848800-ce557880-d19d-11ea-8b69-8774d797d75d.png)

One might argue that the reader should figure this out themselves, but I'm a fan of preventing confusion as much as possible

Reply:
good call, i've added this, 

![image](https://user-images.githubusercontent.com/1134620/89076930-17403500-d336-11ea-8fc5-131e40d2ce5b.png)